media-libs/allegro/allegro-4.4.2-r3.ebuild

```diff
@@ -0,0 +1,111 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-multilib desktop
+
+DESCRIPTION="cross-platform multimedia library"
+HOMEPAGE="https://liballeg.org/"
+SRC_URI="mirror://sourceforge/alleg/${P}.tar.gz"
+
+LICENSE="Allegro MIT GPL-2+ ZLIB"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~mips ~ppc ~ppc64 ~x86"
+IUSE="alsa fbcon jack jpeg opengl oss png svga test vga vorbis X"
+
+RDEPEND="
+	alsa? ( media-libs/alsa-lib[${MULTILIB_USEDEP}] )
+	jack? ( media-sound/jack-audio-connection-kit[${MULTILIB_USEDEP}] )
+	jpeg? ( virtual/jpeg:0[${MULTILIB_USEDEP}] )
+	png? ( media-libs/libpng:0=[${MULTILIB_USEDEP}] )
+	svga? ( media-libs/svgalib[${MULTILIB_USEDEP}] )
+	vorbis? ( media-libs/libvorbis[${MULTILIB_USEDEP}] )
+	X? (
+		x11-libs/libX11[${MULTILIB_USEDEP}]
+		x11-libs/libXcursor[${MULTILIB_USEDEP}]
+		x11-libs/libXext[${MULTILIB_USEDEP}]
+		x11-libs/libXpm[${MULTILIB_USEDEP}]
+		x11-libs/libXt[${MULTILIB_USEDEP}]
+		x11-libs/libXxf86dga[${MULTILIB_USEDEP}]
+		x11-libs/libXxf86vm[${MULTILIB_USEDEP}]
+		opengl? (
+			virtual/glu
+			virtual/opengl
+		)
+	)"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig
+	X? ( x11-base/xorg-proto )"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-shared.patch
+	"${FILESDIR}"/${P}-underlink.patch
+	"${FILESDIR}"/${P}-gentoo.patch
+	"${FILESDIR}"/${P}-rpath.patch
+	"${FILESDIR}"/${P}-Werror-format-security.patch # bug 540470
+	"${FILESDIR}"/${P}-glibc228.patch               # bug 670781
+	"${FILESDIR}"/${P}-GLX_RGBA_FLOAT_BIT_ARB.patch # bug 672858
+)
+
+src_prepare() {
+	cmake-utils_src_prepare
+
+	sed -i \
+		-e "s:allegro-\${ALLEGRO_VERSION}:${PF}:" \
+		docs/CMakeLists.txt || die
+}
+
+src_configure() {
+	# WANT_LINUX_CONSOLE is by default OFF
+	# WANT_EXAMPLES doesn't install anything
+	local mycmakeargs=(
+		-DDOCDIR=share/doc
+		-DINFODIR=share/info
+		-DMANDIR=share/man
+		-DWANT_ALSA=$(usex alsa)
+		-DWANT_EXAMPLES=OFF
+		-DWANT_JACK=$(usex jack)
+		-DWANT_JPGALLEG=$(usex jpeg)
+		-DWANT_LINUX_CONSOLE=OFF
+		-DWANT_LINUX_FBCON=$(usex fbcon)
+		-DWANT_LINUX_SVGALIB=$(usex svga)
+		-DWANT_LINUX_VGA=$(usex vga)
+		-DWANT_LOADPNG=$(usex png)
+		-DWANT_LOGG=$(usex vorbis)
+		-DWANT_OSS=$(usex oss)
+		-DWANT_TESTS=$(usex test)
+		-DWANT_TOOLS=$(usex X)
```

Comment:
should this maybe be native-abi only, if it installs binaries?

Reply:
This part already on multilib_is_native_abi part on installation step.